src/Cache/Cache.php

```diff
@@ -17,6 +22,18 @@ public function __construct(Adapter $adapter)
         $this->adapter = $adapter;
     }
 
+    /**
+     * Toggle case sensitivity of keys inside cache
+     *
+     * @param string $key
+     * @param boolean $value if true, cache keys will be case sensitive
+     * @return mixed
+     */
+    public static function setCaseSensitivity(bool $value)
```

Comment:
Any reason to use static method here?

Reply:
After discussion with Christy we decided to use static method as it allows us to define instance-level configuration.